tests/package.json

```diff
@@ -5,8 +5,10 @@
   "main": "index.js",
   "scripts": {
     "generate-specs": "cd .. && scripts/generate-test-specs.sh",
+    "test-only-with-logs": "mocha  --printlogs -r ts-node/register 'tests/**/*.ts'",
```

Comment:
Something I ran into earlier is that passing`'tests/**/*.ts'` to `mocha` prevents us from running `npm run test <path_to_my_specific_test>`. Is there another way around this (other than hacking `package.json`)?

Reply:
I don't think so, but you can use the --grep 'whatever' trick to select the tests to run

Reply:
@notlesh you can use --grep or add `.only` to a test to execute a specific one

Reply:
I usually issue the mocha command directly when running tests locally. For example, `mocha -r ts-node/register <path_to_my_specific_test>`